src/helper/class-ee-site.php

```diff
@@ -197,6 +198,15 @@ protected function delete_site( $level, $site_url, $site_fs_path, $db_data = []
 			}
 		}
 
+		$volumes_dir   = '/var/lib/docker/volumes';
+		$volume_prefix = \EE\Site\Utils\get_site_prefix( $site_url );
+		$finder        = new Finder();
+		$finder->in( $volumes_dir );
+		$volumes_to_delete = $finder->directories()->name( $volume_prefix . '*' );
```

Comment:
Finding volume prefix and deleting them is unnecessary. Please create volumes using labels.

Example
```
docker volume create \
  --label "org.label-schema.vendor=EasyEngine" \
  --label "io.easyengine.site=one.test" \
  onetest_htdocs
```

Reply:
This command will list the volumes with specific label
```
docker volume ls --filter="label=org.label-schema.vendor=EasyEngine" --filter="label=io.easyengine.site=one.test"
```

Adding the `-q` flag will only display the name
```
docker volume ls --filter="label=org.label-schema.vendor=EasyEngine" --filter="label=io.easyengine.site=one.test" -q
```